content/blog/metrics-at-imperial.md

```diff
@@ -0,0 +1,121 @@
+
+### Cost and Benefit *(or Why are we doing it?)*
+Having zeroed in on the user benefit of a task, another question arises: is it really worth it? Sometimes the priority 
+we should assign to a task is hard to determine, particularly when we have many 'clients' of different kinds - 
+individual researchers of varying seniority and requirements, users of our 'products', large projects spanning multiple 
+organisations - and requests ranging from tiny cosmetic changes to entire applications. 
+
+A 'cost/benefit' metric at the task level might help us to decide which should take priority, and give us visibility of 
+why we have chosen to prioritise the tasks we have. We decided that for our needs, it would be overkill to set a strict 
+formula to calculate a numerical priority value, but that it would be useful to record a checklist for incoming tasks, 
+recording things like:
+
+Costs: 
+
+* Estimated story points (or days effort)
+* Additional effort to fully specify requirements
+* Any additional software or hardware costs
+* Any risks 
+
+Benefits:
+
+* Benefit to user (e.g. major new feature users have been asking for, or just a cosmetic improvement)
+* Urgency (Is this a 'showstopper'?)
+* How many users this will benefit, and for how long
+* Is this something which could be re-used / give future benefit to the team, or other areas of our work?
+
+All these values will be subjective and uncertain, and it may take some time for us to settle on the right checklist to 
+capture. However, having a framework such as this will enable us to start to quantify the priority of our incoming tasks 
+and give visibility to why prioritisation decisions have been made. 
+
+### Communicate Success *(or Wow!)*
+If the ultimate purpose of metrics is to prove our worth, and to enable us to improve, then it's not enough to just 
+record how well we've done - we need to tell someone! 
+
+We'd like to establish an identity for ourselves as a team, rather than just a bunch of individually employed RSEs, 
+and to communicate ways of working together which we have found successful, as well as to let people within DIDE, 
+Imperial College and beyond know how we have solved particular problems. We hope this will be of use to others, either 
+by enabling them to use our applications directly, or by providing some ideas on how they could approach similar 
+problems themselves. Or just by letting people know we exist!
+
+Of course, we could also use the same channels to communicate lessons learned the hard way, or why certain things we 
+tried didn't work, and we would also like to receive feedback and ideas on how we could do things better. 
+
+We're hoping to improve our external communications through talks and presentations, by getting involved in the wider 
+RSE community, and also through this blog. So watch this space!
+
+Do these experiences resonate with any readers in other RSE teams? What metrics do you focus on? Let us know!
+
+**Puns or kilos?*
```

Comment:
I... don't get this :see_no_evil:

Reply:
An appalling pun. Metrics (kilos) at Imperial (pounds/puns). Sorry.